package.json

```diff
@@ -17,26 +17,36 @@
   },
   "dependencies": {
     "@babel/plugin-proposal-decorators": "7.7.4",
+    "@react-native-community/async-storage": "1.7.1",
+    "@types/react-native-dotenv": "0.2.0",
+    "@types/uuid": "3.4.6",
     "axios": "0.19.0",
     "babel-plugin-module-resolver": "3.2.0",
     "date-time-format-timezone": "1.0.21",
+    "dayjs": "1.8.19",
     "jsc-android": "241213.1.0",
+    "path": "0.12.7",
     "pretty-quick": "2.0.1",
     "react": "16.9.0",
     "react-intl": "3.9.1",
     "react-native": "0.61.5",
     "react-native-config": "0.12.0",
+    "react-native-dotenv": "0.2.0",
     "react-native-gesture-handler": "1.5.2",
     "react-navigation": "4.0.10",
     "react-navigation-stack": "1.10.3",
     "react-redux": "7.1.3",
     "redux": "4.0.4",
+    "redux-logger": "3.0.6",
```

Comment:
redux logger shoudl be in dev-dependecies not standard dependecies. Same all typescript stuff like ciustom types etc.